tests/backup/backup_portworx_test.go

```diff
@@ -630,6 +630,8 @@ var _ = Describe("{RestoreEncryptedAndNonEncryptedBackups}", Label(TestCaseLabel
 			log.FailOnError(err, "Fetching px-central-admin ctx")
 			err = ValidateBackupLocation(ctx, BackupOrgID, backupLocationNames[1], BackupLocation1UID)
 			log.FailOnError(err, "backup location %s validation failed", backupLocationNames[1])
+			err = WaitForBackupLocationAddition(ctx, backupLocationNames[1], BackupLocation1UID, BackupOrgID, BackupLocationValidationTimeout, BackupLocationValidationRetryTime)
```

Comment:
Should we wait first and then validate?

Reply:
Backup location is already added , few backup are also taken
After taking backup, we have to validate the encrypted bkp loc and take restore
We are seeing that bkp loc validation takes time and we start restore even before validation is complete.
Hence start validation using "ValidateBackupLocation" and then wait for validation completion using "WaitForBackupLocationAddition"